PostHog/posthog
# fix(queries): Fix test involving timezone property

## Problem

Since timezone_info is calculated based on a field on the model that comes directly from the database, caching it gets problematic. Some tests change the underlying timezone, but timezone_info then becomes stale.
Removing the cached_property decorator exposes some inaccurate tests.

Background: I don't want to put special "refresh this timezone_info" into tests nor have them test wrongly in #18831 

## Changes

- put `@property` instead of `@cached_property`
- adjusts tests so that output matches
- since we likely never rely on this caching outside of tests, there should be *no functional changes* introduced with this PR

## How did you test this code?

- it's tests
- you can check the behavior via the frontend:
   - set your timezone to US/Pacific
   - make some insights query, adjust date_to to something early in the day, like `2023-12-01 05:00`
   - see that the printed SQL query in console queries for `2023-11-30` 
   - thus tests did not reflect real world results
